### PR TITLE
Update gql 0.3.0

### PR DESCRIPTION
**Pull request by KingDarBoja**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Fixes #2 
closes #1 
closes #3

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,3 +45,4 @@ about:
 extra:
   recipe-maintainers:
     - ozagordi
+    - KingDarBoja
```
Comment (beckermr) [inline recipe/meta.yaml]: @KingDarBoja Would you agree to help with the maintenance of this recipe in the future? It is not that much work and you did a great job updating the recipe just now!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ source:
 build:
   noarch: python
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (beckermr) [inline recipe/meta.yaml]: This change will solve the linter complaint.

File: recipe/meta.yaml (hunk 1 above)
Comment (KingDarBoja) [reply recipe/meta.yaml]: No problem mate, go ahead! 👍

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Pull request merged**